README.md

```diff
@@ -207,6 +207,18 @@ if you need information about the registered id.
     }
   }
   ```
+* For remote references, if no version is specified, the latest version of schema is fetched and used from the schema registry.
+  You can also specify `-1` for the same functionality
+```groovy
+schemaRegistry {
+    url = 'http://registry-url:8081'
+    register {
+        subject('avroWithRemoteReferences', '/absolutPath/dependent/path.avsc', "AVRO")
+                .addReference('avroSubject', 'avroSubjectType')
+                .addReference('avroSubjectTwo', 'avroSubjectTwoType', -1)
```

Comment:
Instead of redefining the whole example, could you update one of the reference in the register (and compatibility) examples ?